content/reference/admin/private_locations/configuration/dedicated/index.md

```diff
@@ -0,0 +1,147 @@
+  # Control plane description (optional)
+  description = "my control plane description"
+  # Locations configurations
+  locations = [
+    {
+      # Private location ID, must be prefixed by prl_, only consist of numbers 0-9, 
+      # lowercase letters a-z, and underscores, with a max length of 30 characters
+      id = "prl_private_location_example"
+      # Private location description (optional)
+      description = "Private Location on Dedicated Machines"
+      # Private location type
+      type = "dedicated"
+      ssh {
+        # SSH user
+        user = "gatling"
+        ## SSH private key
+        private-key {
+          # SSH private key path used to secure connections
+          path = "private/key/path"
+          ## SSH private key password (optional, default: none)
+          password = "password"
+           # SSH private key password with an environment variable
+          password = ${?PRIVATE_KEY_PASSWORD}
+        }
+        # SSH port (optional, default: 22)
+        # port = 22
+        # SSH connection timeout (optional, default 15 seconds)
+        # connection-timeout = 15 seconds
+      }
+      # Working directory on hosts, must be executable (optional, default: /tmp)
+      # working-directory = "/tmp
+      
+      # Hosts accessible with the SSH private key, on configured port (hostnames or IP addresses)
+      hosts = [
+        "localhost",
+        "0.0.0.0"
+      ]
+      
+      # Java configuration (following configuration properties are optional)
+      # System properties (optional)
+      system-properties {
+        # ExampleKey = ExampleValue
+      }
+      # Overwrite JAVA_HOME definition (optional)
+      # java-home = "/usr/lib/jvm/zulu"
+      # JVM Options (optional)
+      # Default ones, that can be overriden with precedence:
+      # [
+      #   "-Xmx4G", 
+      #   "-XX:MaxInlineLevel=20", 
+      #   "-XX:MaxTrivialSize=12", 
+      #   "-XX:+IgnoreUnrecognizedVMOptions", 
+      #   "--add-opens=java.base/java.nio=ALL-UNNAMED", 
+      #   "--add-opens=java.base/jdk.internal.misc=ALL-UNNAMED"
+      # ]
+      # Based on your instance configuration, you may want to update Xmx and Xms values.
+      # jvm-options = ["-Xmx4G", "-Xms512M"]
+    }
+  ]
+}
+```
+
+{{< alert warning >}}
+Working directory must be executable
+{{< /alert >}}
+
+## Troubleshooting
+
+### SSH keys
+
+Execute the command `ssh-keygen -t ed25519 -f control-plane` to generate your key pair. _(You have the option to configure a password during this process)_
+
+This will generate the `control-plane` private key and `control-plane.pub` public key in the current directory where the command was executed.
+
+Next, you'll need to configure the authorized keys on the host for a user named, for the example, `gatling`.
+
+To do this, follow these steps:
+
+1. Create the authorized_keys file in the gatling/.ssh directory on the host:
+```bash
+touch gatling/.ssh/authorized_keys
+chmod 600 gatling/.ssh/authorized_keys
+```
```

Comment:
I guess the code block is part of the item in the list.

````suggestion
1. Create the authorized_keys file in the gatling/.ssh directory on the host:
    ```bash
    touch gatling/.ssh/authorized_keys
    chmod 600 gatling/.ssh/authorized_keys
    ```
````